src/main/java/com/hermesworld/ais/galapagos/topics/service/impl/MessagesService.java

```diff
@@ -0,0 +1,54 @@
+package com.hermesworld.ais.galapagos.topics.service.impl;
```

Comment:
Wrong package. Please move to a package `com.hermesworld.ais.galapagos.messages.impl`.